### PR TITLE
Updated "title" and "description" in notebook metadata

### PR DESCRIPTION
**Pull request by tomflexcompute**
According to the SEO consultant,  "title" should be between 30-60 characters (including space) and "description" should be 70-150 characters. The title should also contain "Flexcompute".

Going back and forth with notebook metadata has finalized forced me to make an automation script to handle all the metadata management and updates. From now on changing metadata will be much easier and faster. No more manual opening the notebooks, changing the text, and saving!

### CODE REVIEW

**Comment by tylerflex** [comment]
Great! maybe it would just be easier for me to look at the script and the table of metadata rather than go through each notebook one by one. Should we just include that in the tidy3d-docs repository somewhere? like  in `docs/source/metadata/`?

**Comment by tomflexcompute** [comment]
That's a good idea. Under the metadata folder, metadata.txt file stores the metadata for all notebooks. description_length_checker.py and title_length_checker.py check the length of the title and description for each notebook. update_notebook_metadata.py reads the metadata.txt and updates the notebooks.

File: docs/source/notebooks/metadata/metadata.txt [hunk 1]
```diff
@@ -0,0 +1,464 @@
...
+  },
+  "90OpticalHybrid.ipynb": {
+    "title": "90 Degree Optical Hybrid Modeling in Tidy3D | Flexcompute",
+    "description": "This notebook demonstrates how to model a 90 degree optical hybrid in Tidy3D FDTD.",
+    "keywords": "silicon photonics, PIC, integrated photonics, optical hybrid, MMI, Tidy3D, FDTD",
+    "feature_image": "./img/optical_hybrid_schematic.png"
+  },
+  "AdjointPlugin1Intro.ipynb": {
+    "title": "FDTD Adjoint Optimization Basics | Flexcompute",
+    "description": "This notebook introduces the basics of Jax, automatic differentiation, and adjoint plugin in Tidy3D FDTD.",
```
Comment (tylerflex) [inline docs/source/notebooks/metadata/metadata.txt:16]: "the" adjoint plugin

File: docs/source/notebooks/metadata/metadata.txt [hunk 2]
```diff
@@ -0,0 +1,464 @@
...
+  },
+  "Fitting.ipynb": {
+    "title": "Fitting Dispersive Material Models in Tidy3D | Flexcompute",
+    "description": "This notebook demonstrates how to fit dispersive material models in Tidy3D for FDTD simulations.",
+    "keywords": "dispersive material, fitting, Tidy3D, FDTD",
+    "feature_image": "N/A"
+  },
+  "FocusedApodGC.ipynb": {
+    "title": "Focusing Apodized Grating Coupler | Flexcompute",
+    "description": "This notebook demonstrates a focusing apodized grating coupler to vertically couple light in/out photonic integrated circuits (PICs).",
```
Comment (tylerflex) [inline docs/source/notebooks/metadata/metadata.txt:166]: in/out "of"

File: docs/source/notebooks/metadata/metadata.txt [hunk 3]
```diff
@@ -0,0 +1,464 @@
...
+  },
+  "SelfIntersectingPolyslab.ipynb": {
+    "title": "Modeling Complex Polygon Geometries in Tidy3D | Flexcompute",
+    "description": "This notebook demonstrates how to model complex polygon geometries in Tidy3D for FDTD simulations.",
+    "keywords": "polygon, Tidy3D, FDTD",
+    "feature_image": "N/A"
+  },
+  "Simulation.ipynb": {
+    "title": "First Walkthrough in Tidy3D | Flexcompute",
+    "description": "This notebook focuses on illustrating the basic setup, run, and analysis of a Tidy3D simulation",
```
Comment (tylerflex) [inline docs/source/notebooks/metadata/metadata.txt:358]: "focuses on illustrating" -> "illustrates"

File: docs/source/notebooks/metadata/metadata.txt [hunk 4]
```diff
@@ -0,0 +1,464 @@
...
+  },
+  "YJunction.ipynb": {
+    "title": "Waveguide Y Junction Modeling in Tidy3D | Flexcompute",
+    "description": "This notebook demonstrates how to simulate a waveguide Y junction in Tidy3D FDTD.",
+    "keywords": "Y junction, T branch, Tidy3D, FDTD",
+    "feature_image": "./img/y_junction_schematic.png"
+  },
+  "ZeroCrossTalkTE.ipynb": {
+    "title": "Exceptional Coupling for Crosstalk Reduction | Flexcompute",
+    "description": "This notebook investigates the utilization of anisotropic metamaterial to reduce waveguide crosstalk by leveraging the exceptional coupling.",
```
Comment (tylerflex) [inline docs/source/notebooks/metadata/metadata.txt:454]: "an" anisotropic metamaterial

Comment (tylerflex) [reply docs/source/notebooks/metadata/metadata.txt:454]: "leveraging exceptional coupling" (I think?)

**Review by tylerflex — CHANGES_REQUESTED** [review]
Few edits, but looks good overall! this will be very nice to have.

File: docs/source/notebooks/metadata/metadata.txt (hunk 4 above)
Comment (tomflexcompute) [reply docs/source/notebooks/metadata/metadata.txt]: Thanks for the great catches!

**Pull request merged**